lib/compiler-singlepass/src/codegen_x64.rs

```diff
@@ -1885,9 +1875,10 @@ impl<'a> FuncGen<'a> {
         self.assembler
             .emit_mov(Size::S64, Location::GPR(GPR::RSP), Location::GPR(GPR::RBP));
         // Initialize locals.
+        let local_count = self.local_count();
```

Comment:
Ah, its here because the mutable borrow of `self.assembler` as a first argument to `init_locals` disallows calling methods on `self`.

Could reorder the arguments instead, but it seems more intrusive than just adding a local here.